examples/autorenderhint/src/index.js

```diff
@@ -0,0 +1,114 @@
+'use strict'
+
+const Prism = require('prismjs');
+const Video = require('twilio-video');
+const getSnippet = require('../../util/getsnippet');
+const getRoomCredentials = require('../../util/getroomcredentials');
+const setupBitrateGraph = require('../../util/setupbitrategraph');
+const helpers = require('./helpers');
+const joinRoom = helpers.joinRoom;
+
+const bgImg = document.querySelector('div#bg-img');
+const mediaContainer = document.querySelector('div#media-container');
+const renderDimensionsOption = document.querySelector('select#renderDimensionsOption');
+const videoEl = document.querySelector('video#remotevideo');
+const visibilityToggleButton = document.querySelector('button#visibilityToggleButton');
+const trackIsSwitchedOff = document.querySelector('span#trackIsSwitchedOff');
+let roomP1 = null;
+let stopVideoBitrateGraph = null;
+
+const handleIsSwitchedOff = (isTrackSwitchedOff) => {
+  if(isTrackSwitchedOff) {
+    trackIsSwitchedOff.textContent = 'Off';
```

Comment:
I think 2 seperate buttons - one for `switch on` and one for `switch off` would make more sense.